src/js/views/modal_views.js

```diff
@@ -7,28 +7,55 @@
 
         model: FigureModel,
 
+        initialize: function(options) {
+
+            var self = this;
+            // when dialog is shown, clear and render
+            $("#dpiModal").bind("show.bs.modal", function(){
```

Comment:
space

Reply:
Where? After the ```function()``` ?
When an anonymous function is passed as an argument like this we don't have a space in ```function(){```. I guess this may not be an official style, and I'm not entirely consistent, it is mostly the style I've adopted. I guess we could try and define/adopt a style guide and clean up a whole bunch of these at once.

Reply:
this is more for consistency